frontends/concrete-python/concrete/fhe/compilation/circuit.py

```diff
@@ -249,6 +249,9 @@ def encrypt(
                 encrypted argument(s) for evaluation
         """
 
+        if self.configuration.simulate_encrypt_run_decrypt:
```

Comment:
no need to explain? like "# In simulate_encrypt_run_decrypt:, we skip encryption and decryption, and run will be normal python execution

Reply:
I don't think it's necessary to put on every place it's used. One documentation in the configuration is enough, which I'll add.

Reply:
I think personally that having more comments in the source code is important, but ok